src/test/resources/sql/visit_insert_data.sql

```diff
@@ -1,3 +1,3 @@
-insert into VISIT (ID, APPOINTMENT_START, APPOINTMENT_END, DOCTOR_ID, PATIENT_ID) values (DEFAULT, '2029-12-01 12:00', '2029-12-01 12:30', 2, NULL)
-insert into VISIT (ID, APPOINTMENT_START, APPOINTMENT_END, DOCTOR_ID, PATIENT_ID) values (DEFAULT, '2029-12-01 13:00', '2029-12-01 13:30', 2, 1)
-insert into VISIT (ID, APPOINTMENT_START, APPOINTMENT_END, DOCTOR_ID, PATIENT_ID) values (DEFAULT, '2022-12-01 13:00', '2022-12-01 13:30', 2, 1)
\ No newline at end of file
+insert into VISIT (ID, APPOINTMENT_START, APPOINTMENT_END, DOCTOR_ID, PATIENT_ID) values (DEFAULT, '2029-12-01 12:00:00.000', '2029-12-01 12:30:00.000', 2, NULL)
```

Comment:
a czemu milisekundy tutaj dodawałeś? pluło błąd? :)

Reply:
Tak, to było kombinowanie z niedziałającym query, ale działa to już zostawiłem haha